packages/runtime/test-runtime-utils/src/mocksDataStoreContext.ts

```diff
@@ -75,6 +77,8 @@ export class MockFluidDataStoreContext implements IFluidDataStoreContext {
 	public createProps?: any;
 	public scope: FluidObject = undefined as any;
 
+	public minVersionForCollab: MinimumVersionForCollab = defaultMinVersionForCollab;
```

Comment:
Nit. Double check the link syntax if we keep it.
```suggestion
	/**
	 * Enables setting the minVersionForCollab for the mock.
	 * @remarks
	 * Code will normally use {@link MockFluidDataStoreContext.getMinVersionForCollab | getMinVersionForCollab} to get the value set here.
	 */
	public minVersionForCollab: MinimumVersionForCollab = defaultMinVersionForCollab;
```